strings/truncate_string.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:test/test.dart';
+
+String truncateString(String string,
+    {int maxNumberOfWords, String suffix = '...'}) {
+  return (string.length - 3) < maxNumberOfWords
```

Comment:
- Ternary checks the character length of the string, not the word count as input parameters suggest.
- subtracting 3 from `string.length` seems to try and account for the length of the suffix, but if a different suffix is supplied, it's length may not be 3.
- If the string's number of words is less than the maxNumberOfWords, you won't have performed a truncation, and will not need the suffix/replacement. ("..." at the end implies there is more to the string's content, but this would not be true if nothing is cut off the end)